src/core/issues-notes.js

```diff
@@ -241,10 +241,15 @@ function isLight(rgb) {
  */
 function createLabelsGroup(labels, title, repoURL) {
   const labelsGroup = labels.map(label => createLabel(label, repoURL));
+  const labelName = labels.map(label => label.name);
```

Comment:
I think this will  be "x,y,z"? What we want is "x, y, and z" (i.e., spaces). You could use `ListFormat`, if it's available (supported in Chrome):
 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat

And .reduce() if not supported (Firefox for now).